angular-workspace/projects/ni/nimble-angular/src/directives/card/nimble-card.directive.ts

```diff
@@ -0,0 +1,14 @@
+import { Directive } from '@angular/core';
```

Comment:
We've started putting each component in their own entry point (though all the existing ones are still in the main entry point since it would be a big breaking change to move them all and we haven't decided when/how to stage that work).

So, this should be in its own entry point under `nimble-angular/card`. You can look at some of our newer components as an example if you need to (rich-text editor/viewer, table, or label providers).